src/somd2/config/_config.py

```diff
@@ -195,6 +197,9 @@ def __init__(
         output_directory: str
             Path to a directory to store output files.
 
+        restart: bool
+            Whether to restart from a previous simulation - files found in {output-directory}.
```

Comment:
Can this just say:

`Whether to restart from a previous simulation using the files found in the 'output_directory'`.

The curly braces won't be expanded to anything here and the `output-directory` doesn't match the keyword name.